crates/utils/src/main.rs

```diff
@@ -3,22 +3,44 @@ use cfg_if::cfg_if;
 fn main() {
   cfg_if! {
     if #[cfg(feature = "full")] {
-      use doku::json::{AutoComments, CommentsStyle, Formatting, ObjectsStyle};
-      use lemmy_utils::settings::structs::Settings;
-      let fmt = Formatting {
-        auto_comments: AutoComments::none(),
-        comments_style: CommentsStyle {
-          separator: "#".to_owned(),
-        },
-        objects_style: ObjectsStyle {
-          surround_keys_with_quotes: false,
-          use_comma_as_separator: false,
-        },
-        ..Default::default()
-      };
-      println!("{}", doku::to_json_fmt_val(&fmt, &Settings::default()));
+      println!("{}", config_to_string())
     } else {
-
     }
   }
 }
+
+#[cfg(feature = "full")]
+fn config_to_string() -> String {
+  use doku::json::{AutoComments, CommentsStyle, Formatting, ObjectsStyle};
+  use lemmy_utils::settings::structs::Settings;
+  let fmt = Formatting {
+    auto_comments: AutoComments::none(),
+    comments_style: CommentsStyle {
+      separator: "#".to_owned(),
+    },
+    objects_style: ObjectsStyle {
+      surround_keys_with_quotes: false,
+      use_comma_as_separator: false,
+    },
+    ..Default::default()
+  };
+  doku::to_json_fmt_val(&fmt, &Settings::default())
+}
```

Comment:
This could use a separate feature flag so that doku doesnt get included for normal builds, but probably not worth the effort.